Engine/Client/Src/GraphicalLoad/GraphicalLoad.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type Client
+** File description:
+** GraphicalLoad
+*/
+
+#include <memory>
+#include <string>
+#include <filesystem>
+
+#include "IGraphic.hpp"
+#include "DLLoader.hpp"
+#include "GraphicalLoad.hpp"
+
+#define PATH_GRAPHIC_LIB "Engine/Client/Lib/Graphic/"
+
+static std::string getPathGraphicalLib()
+{
+    std::string path = PATH_GRAPHIC_LIB;
+
+    for (const auto& file : std::filesystem::directory_iterator(path)) {
+        std::string prefix = path + "libgraphic_";
+#ifdef _WIN32
+        std::string suffix = ".dll";
+#else
+        std::string suffix = ".so";
+#endif
```

Comment:
If I remember well I already defined this in the DLLoader file, so maybe you can use those defines. And pleasseee, don't use #ifdef in .cpp files 😭